inc/saiport.h

```diff
@@ -1455,6 +1457,17 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_INTERFACE_TYPE,
 
+    /**
+     * @brief Configure advertised interface type list
+     *
+     * Used when auto negotiation is on. Empty list means all supported values are enabled.
+     *
+     * @type sai_s32_list_t sai_port_interface_type_t
+     * @flags CREATE_AND_SET
+     * @default empty
+     */
+    SAI_PORT_ATTR_ADVERTISED_INTERFACE_TYPE,
+
```

Comment:
Comments added for SAI_PORT_ATTR_INTERFACE_TYPE and SAI_PORT_ATTR_ADVERTISED_INTERFACE_TYPE are the same.  It can't quite tell users when to use which? Thanks.

Reply:
Added comments for all.